README.md

```diff
@@ -227,6 +227,59 @@ If you want a PDB for such a deployment, you can either:
 
 This behavior applies to both integer values (`maxUnavailable: 1`) and percentage values (`maxUnavailable: 25%`). Only deployments with `maxUnavailable: 0` or `maxUnavailable: 0%` will automatically get PDBs created.
 
+### Enabling Eviction Autoscaler for Specific Namespaces
+
+By default, eviction autoscaler is **enabled** for all deployments in the `kube-system` namespace. For deployments in other namespaces, you must explicitly enable the eviction autoscaler by adding the following annotation **to the namespace**:
```

Comment:
So my take here is that we should do all namespaces by deault and they opt out of namespaces
Then in helmcahrt/controller manager therer can be a flag to only opt in by default (could provide namespaces there) . We shoudln't actually make kube-system special.